jsapp/js/formbuild/containers/KoboMatrix.es6

```diff
@@ -96,17 +96,17 @@ class KoboMatrix extends React.Component {
 
   autoName(val, type, ln = false) {
     var names = [];
-    const kobomatrix_list = this.state.kobomatrix_list;
     var data = this.state.data;
 
     if (type === 'column') {
-      data.get('cols').forEach(function(ch){
+      data.get('cols').forEach(function (ch) {
```

Comment:
What's your preference for this kind of situation?
```javascript
someArray.forEach((...) => {
  ...
})
```
or 
```javascript
someArray.forEach(function (...) {
  ...
})
```

Reply:
The preference is the arrow function, just for brevity :)